experimental/dds/tree2/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -577,9 +580,59 @@ describe("SequenceField - Sandwich Rebasing", () => {
 		const moveInverse = invert(move);
 		const undo = tagChange(moveInverse, tag2);
 		const moveRollback = tagRollbackInverse(moveInverse, tag3, tag1);
-		const rebasedUndo = rebaseTagged(undo, moveRollback, move);
+		const rebasedUndo = rebaseOverChanges(undo, [moveRollback, move]);
 		assert.deepEqual(rebasedUndo, undo);
 	});
+
+	it("delete ↷ two inverse inserts", () => {
+		// Given a branch with three changes:
+		// A: Insert x at index 0
+		// B: Insert y at index 0
+		// C: Delete y
+		// This test simulates rebasing C back to the trunk.
+
+		const changeC = tagChange([Mark.delete(1, brand(0))], tag3);
+
+		const rollbackTag2 = mintRevisionTag();
+		const changeB = tagChange([Mark.insert(1, brand(0))], tag2);
+		const inverseB = tagRollbackInverse(invert(changeB), rollbackTag2, tag2);
+
+		const rollbackTag1 = mintRevisionTag();
+		const changeA = tagChange([Mark.insert(1, brand(0))], tag1);
+		const inverseA = tagRollbackInverse(invert(changeA), rollbackTag1, tag1);
+
+		const revInfos: RevisionInfo[] = [
+			{ revision: rollbackTag2, rollbackOf: tag2 },
+			{ revision: rollbackTag1, rollbackOf: tag1 },
+			{ revision: tag1 },
+			{ revision: tag2 },
+			{ revision: tag3 },
```

Comment:
This `{ revision: tag3 }` doesn't seem necessary. It doesn't really bother but I'm curious having it is part of the contract for rebase. If it is I'd like to better understand why, and if not I guess we can remove this line so that we don't risk changing the contract without realizing without the test failing.